build-logic/basics/src/main/kotlin/gradlebuild.repositories.gradle.kts

```diff
@@ -16,7 +16,20 @@
 
 repositories {
     maven {
-        name = "Gradle libs"
-        url = uri("https://repo.gradle.org/gradle/libs")
+        name = "Gradle public repository"
+        url = uri("https://repo.gradle.org/gradle/public")
+        content {
+            includeGroup("net.rubygrapefruit")
+            includeGroup("org.gradle.org.openjdk.jmc")
```

Comment:
You can now use the new version of Gradle profiler (0.17.0-alpha10 on Maven Central) to get rid of that dependency.

Reply:
```suggestion
```